### DISCUSSION TITLE
Target a layer for touch events

### DISCUSSION THREAD

**Discussion by aminerol** [airbnb/lottie-ios#1806] [Q&A]
can we target a specific layer so we can listen for touch events?

**Comment by calda** [comment]
You may be able to use `LottieAnimationView.convert(_ rect: CGRect, toLayerAt keypath: AnimationKeypath?) -> CGRect?` to check if a touch intersects with a specific layer. This is currently only supported for the Main Thread rendering engine though.

**Reply by aminerol** [reply]
Thanks for the quick reply @calda, not sure really how i can achieve that since couldnt find much documentation around that. do you happen to have an example for using `convert` in this specific context? Thank you
